packages/server-admin-ui/src/components/Footer/Footer.js

```diff
@@ -2,38 +2,57 @@ import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import { Link } from 'react-router-dom'
 
+function fetchVessel() {
+  fetch(`/vessel`, {
+    credentials: 'include'
+  })
+    .then(response => response.json())
+    .then(data => {
+      this.setState({ ...data, hasData: true })
+    })
+}
 class Footer extends Component {
+
   constructor(props) {
+
     super(props)
     this.state = {
     }
+    this.fetchVessel = fetchVessel.bind(this)
+  }
+
+  componentDidMount() {
+    this.fetchVessel()
   }
 
   render() {
+    console.log(this)
     return (
       <footer className='app-footer'>
         <span>
           <a href='https://github.com/SignalK/signalk-server-node/'>
             Signal K Server
-          </a>      
+          </a>
         </span>
         {typeof this.props.serverSpecification.server !== 'undefined' && (
           <span>
-            &nbsp; version {this.props.serverSpecification.server.version} 
+            &nbsp; version {this.props.serverSpecification.server.version}
           </span>
         )}
-      {typeof this.props.appStore.serverUpdate !== 'undefined' && (
-        <span>
-          <Link to='/serverConfiguration/update'>
-          &nbsp;(version {this.props.appStore.serverUpdate} is available)
+        {typeof this.props.appStore.serverUpdate !== 'undefined' && (
+          <span>
+            <Link to='/serverConfiguration/update'>
+              &nbsp;(version {this.props.appStore.serverUpdate} is available)
           </Link>
-        </span>
-      )}
+          </span>
+        )}
         {this.props.loginStatus.status === 'loggedIn' && (
           <span className='ml-auto'>
             Logged in as {this.props.loginStatus.username}
           </span>
         )}
+
+        &nbsp;- {this.state.name || this.state.uuid}
```

Comment:
MMSI is better than uuid (shorter, more permanent, better identifier), so insert it into the chain before uuid.

Reply:
Allright, updated.